### ISSUE TITLE
9001 + 9001001 landuse=construction without construction

### ISSUE THREAD

**Issue by maro-21** [osm-fr/osmose-backend#2374]
`landuse=construction without construction` is not an error. Sometimes it is impossible to specify what kind of construction it is.
The same with `building=construction without construction` - I can see that they're building a building, but I don't know what will it be.
Please remove these "errors", because that way we could create thousands of such issues: shop without opening_hours, restaurant without delivery, road without surface etc.

**Comment by Famlam** [comment]
I agree with you, they're annoying (frequently you cannot even see what is being build, residential or retail), but unfortunately these rules result from JOSM and are only imported together with several others:
https://josm.openstreetmap.de/browser/josm/trunk/resources/data/validator/combinations.mapcss#L172

**Comment by maro-21** [comment]
> and are only imported together with several others

Isn't it possible to choose issues to import?
